app/(tabs)/summary/main-summary.tsx

```
@@ -1,15 +1,21 @@
-import {useState} from 'react';
+import {useEffect, useState} from 'react';
 import {useLocalSearchParams} from 'expo-router';
 import {ScrollView, View} from 'react-native';
-import {Button, Text} from 'react-native-paper';
+import {Button, IconButton, Text} from 'react-native-paper';
 
-import {BarChart, Chip, DatePicker, PieChartBar} from '@/components';
+import {barDataItem, pieDataItem} from 'react-native-gifted-charts';
 import {lastDayOfMonth} from 'date-fns';
+import _, {parseInt} from 'lodash';
```

Biome

[error] Do not shadow the global "parseInt" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)